test/test_autograd.py

```diff
@@ -4,14 +4,19 @@
 import math
 import tempfile
 import time
-import torch
 import unittest
 import warnings
 from copy import deepcopy
 from collections import OrderedDict
 from itertools import product
 from operator import mul
 from functools import reduce
+import torch
+
+# TODO: remove this global setting
+# Autograd tests use double as the default dtype
+torch.set_default_dtype(torch.double)
```

Comment:
So, I recall that you noticed that the cuda autograd tests were done with float. Is that still the case after this change?

Reply:
I must have been mistaken. Previously an equivalent import came from common_utils, and this setting also applies to CUDA tensors. There is no functional change here.